bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -198,6 +186,10 @@ struct task_struct {
 	int pid;
 } __attribute__((preserve_access_index));
 
+struct task_struct___t {
+	unsigned int tgid;
+};
+
```

Comment:
You don't need to create an new struct - this wouldn't work anyway as tgid isn't the first field (that's state)